types.ts

```diff
@@ -426,12 +679,37 @@ export interface PackVersionDefinition {
    * explicit connection is specified by the user.
    */
   systemConnectionAuthentication?: SystemAuthentication;
+  /**
+   * Any domain(s) to which this pack makes fetcher requests. The domains this pack connects to must be
+   * declared up front here, both to clearly communicate to users what a pack is capable of connecting to,
+   * and for security reasons. These network domains are enforced at execution time: any fetcher request
+   * to a domain not listed here will be rejected.
+   *
+   * Only one network domain is allowed by default. If your pack has needs to connect to multiple domains
+   * contact Coda support for approval.
+   */
   networkDomains?: string[];
 
   // User-facing components
+
+  /**
+   * @deprecated
+   */
   formulaNamespace?: string; // TODO: @alan-fang remove
+  /**
+   * Definitions of this pack's formulas. See {@link Formula}.
```

Comment:
Perhaps note that this is where buttons are also defined, since they are just formulas with isAction: true?

Reply:
Done.